OSDOCS-5604 Removing UnhealthyPodEvictionPolicy from featureset (#63072)
openshift/openshift-docs: modules/pod-disruption-eviction-policy.adoc

IfHealthyBudget:: Running pods that are not yet healthy can be evicted only if the guarded application is not disrupted.

AlwaysAllow:: Running pods that are not yet healthy can be evicted regardless of whether the criteria in the pod disruption budget is met. This policy can help evict malfunctioning applications, such as ones with pods stuck in the `CrashLoopBackOff` state or failing to report the `Ready` status.
+
[NOTE]
====

> It is recommended to set `AlwaysAllow` to your PodDisruptionBudgets to support eviction of misbehaving applications during a node drain.

Reviewer: IMO I would add even the second part: `The default behavior is to wait for the application pods to become healthy before the drain can proceed.` or hint it can potentially have negative implications for the application/

Author: Added, thank you!

> It is recommended to set `AlwaysAllow` to your PodDisruptionBudgets to support eviction of misbehaving applications during a node drain. The default behavior is to wait for the application pods to become healthy before the drain can proceed.

Reviewer 2: ```suggestion
It is recommended to set the `unhealthyPodEvictionPolicy` attribute to `AlwaysAllow` in the `PodDisruptionBudget` object to support the eviction of misbehaving applications during a node drain. The default behavior is to wait for the application pods to become healthy before the drain can proceed.
```

Author: Made the suggested change.

> It is recommended to set the `unhealthyPodEvictionPolicy` attribute to `AlwaysAllow` in the `PodDisruptionBudget` object to support the eviction of misbehaving applications during a node drain. The default behavior is to wait for the application pods to become healthy before the drain can proceed.

Author: Changed, thank you!
